src/notifications/datamodel/models.py

```diff
@@ -21,6 +22,11 @@ class Kanaal(models.Model):
         _('Documentatie link'), blank=True,
         help_text=_('URL naar documentatie.'),
     )
+    filters = ArrayField(
+        models.CharField(max_length=100),
+        verbose_name=_("filters"), null=True,
```

Comment:
I'd remove the `null=True` and replace it with `blank=True, default=list` - this makes the field optional and if you don't fill it in, an empty list will be set. That way, we consistently have a `list` as datatype and not sometimes `None`

Reply:
Thank you! Fixed